MuxExoPlayer/src/r2_10_6/java/com/mux/stats/sdk/muxstats/MuxStatsExoPlayer.java

```diff
@@ -409,6 +409,10 @@ public void onPlayerStateChanged(boolean playWhenReady, int playbackState) {
         }
         break;
       case Player.STATE_IDLE:
+        if (state == PlayerState.PLAYING || state == PlayerState.PLAYING) {
```

Comment:
This is checking the same thing twice... is there a reason?

Reply:
you are right, this is a typo, one should be PLAY